server/src/main/java/com/linecorp/centraldogma/server/internal/admin/model/Role.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2017 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.centraldogma.server.internal.admin.model;
+
+import static java.util.Objects.requireNonNull;
+
+import com.linecorp.centraldogma.server.internal.admin.authentication.User;
+
+/**
+ * Roles of a {@link User} in a project.
+ */
+public enum Role {
+    OWNER,
```

Comment:
- PROJECT_OWNER? (or is it REPOSITORY_OWNER?)
- Should we add ADMINISTRATOR here?

Reply:
How about changing `Role` to `ProjectRole`?

Reply:
Regarding `ADMINISTRATOR`, I think it would be better to think two kinds of user roles separately for now, one is system roles like `administrator` and `general user`, and the other one is project roles like `owner` and `member`. I will add it if necessary in the following PR. :-)

Reply:
SGTM